apps/wallet/tailwind.config.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
             },
             borderRadius: {
                 '2lg': '0.625rem',
+                15: '0.9375rem',
```

Comment:
Border radius values aren't generally numeric, so I'd just name this something else, or use `rounded-2xl` which is 1px off (I've pushed us to align sizing units with tailwind more too)

Reply:
Got it I will switch to a tailwind `rounded-2xl`

Reply:
done